test/e2e/generate.test.ts

```diff
@@ -33,11 +38,9 @@ describe('ipx provider', () => {
         "/_ipx/s_300x300/images/colors.jpg",
         "/_ipx/s_300x300/images/everest.jpg",
         "/_ipx/s_300x300/images/tacos.svg",
-        "/_ipx/s_300x300/unsplash/photo-1606112219348-204d7d8b94ee",
```

Comment:
@pi0 shouldn't we be generating files for remote sources as well when prerendering?

Reply:
found the issue alias was not being passed